pkg/generate/app/pipeline.go

```diff
@@ -91,7 +91,7 @@ func (p *Pipeline) NeedsDeployment(env Environment, name string) error {
 // Objects converts all the components in the pipeline into runtime objects
 func (p *Pipeline) Objects(accept, objectAccept Acceptor) (Objects, error) {
 	objects := Objects{}
-	if p.InputImage != nil && p.InputImage.AsImageStream && accept.Accept(p.InputImage) {
+	if p.InputImage != nil && accept.Accept(p.InputImage) {
```

Comment:
The flags above didn't change, I just packed them into the ImageRef creation. This check here is what prevents our input stream to be created. It doesn't make sense anyway, we have use for Images and ImageStreams down the way, namely https://github.com/openshift/origin/blob/3c874cc03c3473bea4bbd0f75d0e18a7766ee97c/pkg/generate/app/app.go#L252